lib/htmlValidator.js

```diff
@@ -48,29 +51,38 @@ module.exports = function(app, callback) {
 
         validatorProcess.stdout.on('data', (data) => {
           if (`${data}`.includes('Initialization complete')) {
+            validatorDisabled = false;
+            clearTimeout(validatorTimeout);
             resolve();
           }
         });
 
         validatorProcess.stderr.on('data', (data) => {
           if (`${data}`.includes('No Java runtime present, requesting install.')) {
-            console.error('error: No Java runtime detected. Check to make sure Java is installed and in your path.');
+            console.error(('There was an error in authentication. Please make sure to have Java installed.').red);
+            validatorDisabled = true;
             reject();
           }
-          else {
```

Comment:
Alex and I have discussed this and we're in agreement this else should be left out.